cypress/e2e/5-ndv.cy.ts

```diff
@@ -617,6 +617,42 @@ describe('NDV', () => {
 			// Sinse code tool require alphanumeric tool name it would also show an error(2 errors, 1 for each tool node)
 			cy.get('[class*=hasIssues]').should('have.length', 3);
 		});
+
+		it('should have the floating nodes in correct order', () => {
+			cy.createFixtureWorkflow('Floating_Nodes.json', 'Floating Nodes');
+
+			cy.ifCanvasVersion(
+				() => {},
+				() => {
+					// Needed in V2 as all nodes remain selected when clicking on a selected node
+					workflowPage.actions.deselectAll();
+				},
+			);
```

Comment:
It would probably make sense to add this to `createFixtureWorkflow`, since we're manually deselecting nodes like this on multiple places 🤔 But it's definitely out of scope of this PR